stock_putaway_product_form/__manifest__.py

```diff
@@ -0,0 +1,13 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Putaway strategies on product form view",
+    "summary": "Edit putaway strategies directly from the product form view",
+    "version": "12.0.1.0.0",
+    "category": "Inventory",
+    "website": "https://www.akretion.com",
+    "author": "Akretion"
```

Comment:
```suggestion
    "author": "Akretion,"
```

Reply:
@kevinkhao You didn't add the comma (Travis is failing for that reason)